frontend/src/main/pages/HomePage.js

```diff
@@ -25,6 +26,21 @@ export default function HomePage() {
         url: "/api/commons/all"
       }
     );
+  const onSuccess = (commons) => {
+    // Stryker disable next-line all : hard to get variable
+    var existed = new Boolean(false);
+    for(let i = 0; i < commonsJoined.length; i++ ){
+      if(commonsJoined[i].id == commons.id){
+        existed = true;
+      }
+    }
+    if(existed == true){
```

Comment:
nitpick, but this can be simplified. also we should use `===` whenever possible.
```suggestion
    if(existed){
```

Reply:
Actually if(existed) does not work. It would always go to the first conditional. I changed it to if(existed === true)